main.go

```diff
@@ -227,7 +230,9 @@ func createTable(db *sql.DB) error {
 		user_agent TEXT,
 		daily_user_type_native BIGINT,
 		daily_user_type_bridged BIGINT,
-		daily_user_type_guest BIGINT
+		daily_user_type_guest BIGINT,
+		python_version TEXT
```

Comment:
Just to check, are we happy with the implied default of this value, which i think is NULL, in our dashboards and similar?